### PR TITLE
Azure CLI Support for Email Services Control Plane CRUD operations

### PR DESCRIPTION
**Pull request by ddouglas-msft**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
```
Comment (kairu-ms) [inline src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py:7]: Why the generated folder name is `emailcommunication`? It shoudl be `communication\email` if directly generated by aaz-dev-tools. Please do not manually change the folder of `aaz`, it should be directly generated from `aaz-dev-tools`.

File: src/communication/azext_communication/manual/_help.py [hunk 2]
```diff
@@ -575,3 +575,165 @@
         text: |-
                az communication email send --sender "NoReply@contoso.com" --subject "Contoso Update" --to "user1@user1-domain.com" "user2@user2-domain.com" --text "Hello valued client. There is an update."
 """
+
+helps['communication email create'] = """
```
Comment (kairu-ms) [inline src/communication/azext_communication/manual/_help.py:579]: The help message is not required adding in `_help.py` for the `aaz` commands. They are already contained in the the aaz commands.

File: src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py (hunk 1 above)
Comment (ddouglas-msft) [reply src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py]: Thank you. This has been updated

File: src/communication/azext_communication/manual/_help.py (hunk 2 above)
Comment (ddouglas-msft) [reply src/communication/azext_communication/manual/_help.py]: Thank you. This has been updated

File: src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py (hunk 1 above)
Comment (ddouglas-msft) [reply src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py]: Hello @kairu-ms. Please let me know if you have any further questions or concerns. Thank you

Comment (ddouglas-msft) [reply src/communication/azext_communication/aaz/latest/emailcommunication/__cmd_group.py]: Hello @AllyW 
Please let me know if you need additional information. Thank you

File: src/communication/azext_communication/manual/_help.py (hunk 2 above)
Comment (ddouglas-msft) [reply src/communication/azext_communication/manual/_help.py]: Hello @kairu-ms. Thank you for your time. I was wondering if you could check this PR again. Thank you

**Pull request closed without merging**